cfme/services/catalogs/orchestration_template.py

```diff
@@ -114,6 +125,25 @@ def delete(self):
         flash.assert_success_message('Orchestration Template "%s" was deleted.' %
                                      self.template_name)
 
+    def edit_tags(self, tag, value):
+        sel.force_navigate("select_template", context={"template_type": self.template_type,
+                                                       'template_name': self.template_name})
```

Comment:
@satyajitbulage You should use the `cfme.web_ui.mixins` instead of defining yet another tag edit form. Or if the class can support `load_details(refresh)` method, then you can `from cfme.common import TaggableMixin` and inherit `TaggableMixin` to get methods that manipulate with tags.